docs/usage/getting-started.md

```diff
@@ -74,27 +74,49 @@ kind create cluster
 
 [OIDC (OpenID Connect)](https://openid.net/connect/) is a popular authentication often available in your company or major cloud providers. For local purposes we will run our own OIDC provider to handle authentication. We are going to use ORY Hydra for that. First download and extract the binary release from [here](https://github.com/ory/hydra/releases/download/v1.9.1/hydra_1.9.1-sqlite_linux_64bit.tar.gz).
 
+**Linux**
+
 ```bash
 curl -L "https://github.com/ory/hydra/releases/download/v1.9.1/hydra_1.9.1-sqlite_linux_64bit.tar.gz" | tar -xzf - -C tmp/bin hydra
 ```
 
+**MacOS**
+
+```bash
+curl -L "https://github.com/ory/hydra/releases/download/v1.9.1/hydra_1.9.1-sqlite_macos_64bit.tar.gz" | tar -xzf - -C /tmp/bin hydra
+```
+
 The configuration file for `hydra` is present in [`configs/hydra.yaml`](/configuration/examples/local/configs/hydra.yaml).
 
 ```yaml mdox-exec="cat configuration/examples/local/configs/hydra.yaml"
 strategies:
   access_token: jwt
 urls:
   self:
-    issuer: http://172.17.0.1:4444/
+    issuer: http://docker.for.mac.localhost:4444/
+
```

Comment:
I think this extra line isn't necessary
```suggestion
```

Reply:
ah yes, indeed, CI is failing because of this

Reply:
TIL about mdox! Fixed.